README.md

```diff
@@ -10,6 +10,58 @@ The Shipyard project provides tooling for creating K8s clusters with [kind](K8s
 - [go 1.12] with [$GOPATH configured]
 - [docker]
 
+## Usage
+To use shipyard for your project, it's easiest to use dapper and make.
```

Comment:
```suggestion
To use Shipyard for your project, it's easiest to use Dapper and Make.
```

Reply:
dapper and make are normally called in low-case from the cmdline, for those may be it's better to use lowcase?

Reply:
I think if we want to use literal commands, we should use `make` or `dapper`. If we are talking about the tools themselves, I think they are proper nouns and should be Capitalized.

Reply:
I agree that there are a lot of cases of them lowercase though, so definitely a non-blocker. Wikipedia uses it as I described I think. https://en.wikipedia.org/wiki/Make_(software)